index.d.ts

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line strict
```

Comment:
suggestion: can you disable this via our ESLint config?

Reply:
Fixed